source/layouts/layout.html.haml

```diff
@@ -7,8 +7,8 @@
 - newer_version_available = File.exist?("source/#{language_code}/#{$last_version}")
 
 !!! 5
-%html
-  %head{ lang: current_page.data.language }
+%html{ lang: current_page.data.language }
```

Comment:
Just to be clear here, this change allows browsers to recognize that while most of the document uses the current language, the changelog being previewed inside the page doesn't? So hypothetically it would offer useful translation just for that part of the page?

Reply:
Yes, precisely! Sorry about the lack of _why_ in the PR description.